content/rc/administration/setup/customize-flexible-plan.md

```diff
@@ -0,0 +1,123 @@
+---
+Title: Create a Flexible subscription
+description:
+weight: 30
+alwaysopen: false
+categories: ["RC"]
+aliases: /rc/administration/customize-pro/ 
+         /rc/administration/customize-pro.md 
+         /rc/administration/customize-flexible-plan/
+---
+
+When you create a subscription with a [Flexible plan]({{< relref "/rc/administration/setup/#subscription-plans" >}}), you customize it to your business needs.
+
+This means you:
+
+1. Define deployment settings for your subscription, such as cloud provider and network settings.
+
+1. Create at least one database to determine the size of the underlying cluster.
+
+1. Review the cost estimate for your selections and define payment before creating the subscription.
+
+This article describes the Create Custom Subscription screen, which uses tabs to walk you through each step. The following sections summarize the avilable options.
+
+## Setup tab
+
+The Setup tab specifies general settings for the deployment of your subscription.
+
+{{<image filename="images/rc/flexible-create-setup.png" width="75%" alt="The Setup tab of the Create Custom Subscription screen." >}}{{< /image >}}
+
+There are three sections on this tab:
+
+- [General settings](#general-settings) include the name of your subscription and the memory storage.
+- [Cloud details](#cloud-details) identify the cloud provider, region, and cloud account (AWS only).
+- [Advanced options](#advanced-options) vary according to cloud provider.
+
+### General settings {#general-settings}
+
+| _General setting_ | _Description_ |
+|:---------|:-----------|
+| **Subscription&nbsp;Name** | A custom name for your subscription (_required_) |
+| **Runs On**| Defines whether your databases are stored only in memory (RAM) or are split between memory and Flash storage (RAM+Flash).  See [Redis on Flash]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}})|
+
+### Cloud details {#cloud-details}
+
+| _Cloud detail setting_ | _Description_ |
+|:---------|:-----------|
+| **Cloud Vendor** | The public cloud provider |
+| **Region(s)** | The cloud region for your subscription |
+| **Multi-AZ** | Whether in-memory data is replicated to another zone within the deployment region. For best results, choose a region with at least three availability zones.|
+| **Cloud Account** | (_AWS_) Credentials allowing Redis Cloud to be deployed to an existing AWS account.  See [Create and Edit a Cloud Account]({{< relref "/rc/how-to/view-edit-cloud-account.md" >}})|
+
+### Advanced options {#advanced-options}
+
+| _Advanced option_ | _Description_ |
+|:---------|:-----------|
+| **Deployment CIDR** | The [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) subnet address for your deployment. Must not overlap other addresses used with your subscription.|
```

Comment:
It would be great to add CIDR to the glossary and link that term here.

Reply:
So noted.